src/frontend/scss/objects/_course_glimpses.scss

```diff
@@ -111,11 +111,11 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     display: flex;
     position: absolute;
     top: -1.1rem;
-    left: 0;
+    right: 0;
     flex-direction: row;
     align-items: center;
-    border-top-left-radius: 0;
-    border-bottom-left-radius: 0;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
```

Comment:
There could be quickly an overlap.

<img width="307" alt="Capture d’écran 2021-08-09 à 10 39 44" src="https://user-images.githubusercontent.com/9265241/128679819-64b66a6a-5cb5-473c-9fc4-4af5d211c869.png">

Reply:
True... and the code may be longer too... should I lower the code or truncate the icon label?

Reply:
As a quick solution, I think truncate the icon label is the best choice.

Reply:
I proposed something...

Reply:
@sveetch you might want to take a look at the css :blush:

Reply:
This is more about layout than CSS itself. You putted the reference id just to the left of the category, if category is longer than in your preview, it will go on two lines or more and will go over title to mask it.

You should have put the reference elsewhere like under the title.

Reply:
You edited `category_glimpse.scss` so course glimpses rendered through Django are well displayed but glimpses rendered through React are not well rendered because they didn't use category glimpse.


<img width="297" alt="Capture d’écran 2021-08-09 à 16 28 18" src="https://user-images.githubusercontent.com/9265241/128722950-a953f1e5-527a-465f-afe3-8a2fea733511.png">

Reply:
@jbpenrath @sveetch So your comments have discouraged me from pursuing this option. I have pushed a new version to display the code below the organization with a "barcode" icon to illustrate it. WDYT?

Reply:
@sampaccoud i thought about a different approach, making it absolute in the top left glimpse corner like this:

![reference_sample](https://user-images.githubusercontent.com/1572165/128768540-e8533ff0-2de3-4665-b84e-22e851127107.png)

Here is the sample element CSS (you will have to use the right variable for color, radius, etc..):

```
.course-glimpse__reference {
    position: absolute;
    top: 0;
    left: 0;
    z-index: 4;
    padding: 0.2rem 0.4rem;
    background: white;
    border-top-left-radius: .5rem;
    border-bottom-right-radius: .5rem;
    color: black;
}
```

Reply:
From my point of view , barcode is fine.
I'll just use red color and same font-size like the organization to group these information visually. (like a metadata section).

Reply:
After brainstorming and testing several options with @jbpenrath we decided on:

<img src='https://user-images.githubusercontent.com/1427165/128840904-79abefcc-7b11-4679-98eb-c214035dce93.png' width="350"/>